modules/transcription/transcriber.js

```diff
@@ -85,7 +85,7 @@ Transcriber.prototype.stop = function stop(callback) {
     console.log('stopping recording and sending audio files');
     this.audioRecorder.stop();
 
-    // and send all recorded audio the the transcription service
+    // and send all recorded audio the transcription service
```

Comment:
should be - `and send all recorded audio to the transcription service.`

Reply:
Hi @jallamsetty1 I just did